### ISSUE TITLE
I always check pull requests before issues.

### ISSUE THREAD

**Issue by carls** [rebol/rebol#42]
Just thought I would mention: I always check pull requests before issues, so it is not necessary to add an issue along with the pull... unless you think there really is an issue.

**Comment by earl** [comment]
Github's pull requests are handled as "issues with code attached". That's why for each pull request, an issue is always created alongside it.

**Comment by carls** [comment]
Yes... the reason I mention it is to suggest people actually contribute fixes/code, as that will be my main priority over issues that have no code, and thus require that someone else must take the time to write it.  We all have our wish lists, but without code, they just remain wishes.
